web/src/pages/tasks/[taskUrlSlug].tsx

```diff
@@ -207,3 +207,14 @@ export const getStaticProps = async ({
 };
 
 export default TaskPage;
+
+export const getUnlockedBy = (
```

Comment:
it's typically an anti-pattern to export helper functions from components into other components - keeping logic like this separate from React components can help with refactoring. I think this should go in utils or something if it's going to be imported into other components

Reply:
however, I wonder if this functionality (filter the "unlocked by" tasks to only those that are uncompleted) should really just be logic inside of the UnlockedBy component, instead of a helper function that gets called by every component just to pass into <UnlockedBy />